app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
If you leave this without validations for first_name and last_name, if you try to create an User without any of those fields, it with explode and you will not be able to catch the error, because it is comming from the database.
In the database, you asked for `null: false`, and that is correct but we should add also that validation in the model.
Rails by itself has a lot of errors handling.
Check this link and add the validation of `presence`: https://edgeguides.rubyonrails.org/active_record_validations.html#presence